flink-core/src/main/java/org/apache/flink/api/common/operators/base/ReduceOperatorBase.java

```diff
@@ -52,6 +52,30 @@
  */
 public class ReduceOperatorBase<T, FT extends ReduceFunction<T>> extends SingleInputOperator<T, T, FT> {
 
+	/**
+	 * An enumeration of hints, optionally usable to tell the system exactly how to execute the reduce.
+	 */
+	public enum ReduceHint {
```

Comment:
Should be renamed to `CombineHint` and all corresponding methods and member variables as well.